lib/python/rose/run_source_vc.py

```diff
@@ -43,18 +43,23 @@ def write_source_vc_info(run_source_dir, output=None, popen=None):
         handle = open(output, "wb")
     environ = dict(os.environ)
     environ["LANG"] = "C"
-    for vcs, cmds in [("svn", ["info", "status", "diff"]),
-                      ("git", ["describe", "status", "diff"])]:
+    for vcs, args_list in [
+            ("svn", [
+                ["info", "--non-interactive"],
```

Comment:
I take it these "--non-interactive" arguments are disabling the prompt that was causing the problem?